src/pynxtools/testing/nexus_conversion.py

```diff
@@ -241,17 +241,25 @@ def load_logs(
         def compare_logs(gen_lines: list[str], ref_lines: list[str]) -> None:
             """Compare log lines, ignoring specific differences."""
 
+            def get_section_ignore_lines(line: str) -> list[str]:
+                """Return ignore lines for a section if the line starts with the section."""
+                section = line.rsplit(SECTION_SEPARATOR, 1)[-1].strip()
+                for key, ignore_lines in IGNORE_SECTIONS.items():
+                    if section.startswith(key):
+                        return ignore_lines
+
+                return []
```

Comment:
Can you add a simple example about `ignore_line` and `ignore_section` in the testing framework docs? I had a hard time using them.